### PR TITLE
Add check_version on gravitee

### PR DESCRIPTION
**Pull request by francois2metz**

### CODE REVIEW

File: graviteeio-apim-rest-api/check_version.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+NEXT_VERSION=$(curl \
+  --silent          \
+  https://api.github.com/repos/gravitee-io/gravitee-api-management/tags | jq --raw-output '.[].name' | grep -v '-' | head -1)
```
Comment (jonathanperret) [inline graviteeio-apim-rest-api/check_version.sh]:
```suggestion
  https://api.github.com/repos/gravitee-io/gravitee-api-management/tags | jq --raw-output 'map(select(.name | contains("-") | not))[0].name')
```

File: graviteeio-apim-rest-api/check_version.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+jq --null-input \
+  --arg version "$GRAVITEE_VERSION" \
+  --arg next "$NEXT_VERSION" \
+  '{type: "software_version", software: "gravitee apim", version: $version, available_version: $next}'
```
Comment (jonathanperret) [inline graviteeio-apim-rest-api/check_version.sh]:
```suggestion
jq --null-input --compact-output \
  --arg version "$GRAVITEE_VERSION" \
  --arg available_version "$NEXT_VERSION" \
  '{type: "software_version", software: "gravitee apim", $version, $available_version}'
```

File: graviteeio-apim-rest-api/check_version.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+NEXT_VERSION=$(curl \
+  --silent          \
+  https://api.github.com/repos/gravitee-io/gravitee-api-management/tags | jq --raw-output '.[].name' | grep -v '-' | head -1)
+
+jq --null-input \
```
Comment (jonathanperret) [inline graviteeio-apim-rest-api/check_version.sh]:
```suggestion
jq --null-input --compact-output \
```

**Review by mickaelalibert — APPROVED** [review]
LGTM :rocket:

**Pull request merged**